pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1557,18 +1558,46 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     return JSON.parse(stdout).SPNetworkDataType;
   }
 
+  protected async installContainerdShims() {
+    // shims bundled with the app itself
+    const bundleDir = path.join(paths.resources, 'linux', 'internal');
+    // shims managed by the user
+    const shimDir = path.join(paths.appHome, 'containerd-shims');
```

Comment:
This should be a property on `paths` to reflect the changes to `rdctl`.

Reply:
Yeah, already done this locally.